### DISCUSSION TITLE
How to request removal of "Regional" channels from "Country" playlist?

### DISCUSSION THREAD

**Discussion by eurohunter** [iptv-org/community#330] [Q&A] [answered]
I downloaded a playlist grouped by country (for Albania), and there are several channels that aren't Albanian channels (about half of them). How can we request these "regional" channels be removed from a country playlist? Is there a categorization issue that needs to be fixed?

A full list of channels that do not belong under the "country" playlist of Albania is as follows:
-Al Jazeera Documentary (All in Arabic, Not working)
-ARTE 
-Azerbaijan News TV
-BBC World News (Not Working)
-Bloomberg TV EMEA
-Bloomberg TV Europe
-CNBC Europe
-CNN International Germany (not working)
-Europe by Satellite 
-Europe by Satellite Plus
-EWTN Europe
-Fashion TV
-Galicia TV Europa
-NTV Europe
-RT en Espanol 
-Three Angels
-TV3CAT
-TV5Monde Europe
-WeatherSpy Europe
-PTP (Russian channel)

**Comment by BellezaEmporium** [comment, marked as answer]
I believe those are 'international' channels and appear there. Though I might be wrong.

@freearhey if you have the answer.

**Reply by freearhey** [reply]
International channels after patch #11372 are placed in a separate file `countries/int.m3u`. In this case, most channels have Europe as the broadcast area (for example, https://iptv-org.github.io/channel?id=AzerbaijanNewsTV.se), so the fact that they are in the Albanian playlist is not a mistake.

**Reply by eurohunter** [reply]
Ok, thank you for the clarification. Then, I believe that some international channels would benefit from the recategorization of their broadcast areas. How do I go about requesting such changes linked to broadcast areas?

For instance, looking at the various international channels found in the "Albania" playlist and how they are currently shown in the database, I suggest the following:

- ARTE: This channel works only in certain countries: France and Germany. Perhaps this channel's broadcast area could be categorized under these two countries ("c/FR;c/DE" or any other countries where it might be available) as opposed to all of Europe under r/EUR. It is currently not allowed to stream in Albania (or the Netherlands, tested with VPN).

- Azerbaijan News TV: This channel is in the Azerbaijani language. Also, the link works in the US as well as Japan (tested with VPN). Perhaps it would be better if the broadcast area of this channel would be c/AZ or c/int (still not sure what qualifies a channel to be added to the "c/int" list) as opposed to r/EUR. There doesn't seem to be a regional limitation for this channel.

- Fashion TV: This channel works all over the world. It would now be better if it has a broadcast area under "c/int". 

- Al Jazeera Documentary: This channel's broadcast areas are currently "r/ARAB;r/EUR". However, this link says that "this channel is not available in your region". I tested from Albania as well as with a Netherlands VPN). As such, it would be better if the broadcast area for this channel would only be "r/ARAB".

- RT en Espanol: This channel is accessible worldwide (tested with VPN in USA, Japan, and Netherlands). Its broadcast area could either be classified under "c/int" or under Spanish-speaking regions/countries as opposed to "r/EUR;r/SOUTHAM;c/US". Perhaps the "r/EUR" could be replaced with "c/ES". 

- 3ABN International: This religious channel's broadcast area could now be "c/int" as opposed to "r/EUR;r/AFR;r/SEA;r/OCE".

- TV3 Cat: This is a Catalan channel in the Catalan language. This channel's broadcast area would be better if it is "s/ES-CT" as opposed to "r/EUR". There is no indication that this channel is intended for the entire European audience.

- WeatherSpy: This channel seems to be the same worldwide. There is no distinction among regions. I am looking at the three available streams from the US, Europe, and India, and they are all showing the same show. Should this channel be listed only once as "WeatherSpy" under "c/int" as opposed to different regions and countries?

- Galicia TV Europa: This Galician/Spanish channel would be better suited under "s/ES-GA" as opposed to "r/EUR" because it is in the Galician language and is intended for the Galician/Spanish audience, not for the entire European audience. 

- NTV Europe: This Bangladeshi channel would be more suitable under "c/UK" as opposed to "r/EUR" because it is based in London and all the commercials are for the UK audience (prices shown in Pounds).

- RTR Planeta: This Russian channel would now be more suitable under "c/int" as opposed to r/EUR" because it is available worldwide and the same link is available for everyone.

**Reply by freearhey** [reply]
@eurohunter You can publish all of these requests via this form: https://github.com/iptv-org/database/issues/new?assignees=&labels=channels%3Aedit&template=channels_edit.yml&title=Edit%3A+

**Reply by eurohunter** [reply]
Ok will do. Thank you!

**Reply by fershady** [reply]
you can also get the raw link. https://raw.githubusercontent.com/iptv-org/iptv/master/streams/al.m3u

**Reply by eurohunter** [reply]
> you can also get the raw link. https://raw.githubusercontent.com/iptv-org/iptv/master/streams/al.m3u

thank you fershady. I wasn't aware of raw links. this is very helpful.
